curvefs/conf/client.conf

```diff
@@ -57,7 +56,7 @@ extentManager.preAllocSize=65536
 
 #### brpc
 # close socket after defer.close.second
-rpc.defer.close.second=600
+rpc.defer.close.second=1
```

Comment:
defer set to a long time for reuse, Will this be disconnected under abnormal conditions?

Reply:
it's not, there is an atomic variable that stores currentWorkingMds, and only when RPC succeeded atomic get updated.

so, next `AllocS3ChunkId` will send RPC to power down MDS firstly, because no one succeeds sending RPC to MDS and currentWorkingMds still stores the power down MDS